etc/tfw_frang.conf

```diff
@@ -0,0 +1,45 @@
+#
+# Tempesta FW configuration: "frang" classifier.
+#
+# "include" directives are not supported yet, so paste the configuration to
+# the main configuration file.
+
+# TAG: frang_limits
+#
+# The section containing static limits for the classifier.
+#
+# Syntax:
+#   frang_limits {
+#	request_rate NUM;
+#	request_burst NUM;
+#	new_connection_rate NUM;
+#	new_connection_burst NUM;
+#	concurrent_connections NUM;
+#	http_uri_len NUM;
+#	http_field_len NUM;
```

Comment:
Actually, the correct term is `field`, or `header field`. `Header` refer to the whole header that consist of multiple header fields.